README.md

```diff
@@ -1 +1,201 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Metrics-F1 Driver Stats <a name="about-project"></a>
+
+> In this project, I build a F1 Driver Stats app using react and redux. The app style is based on the Original design idea by Nelson Sakwa on Behance
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> This project uses the following stack :
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/">JavaScript</a></li>
+    <li><a href="https://reactnative.dev/">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+    <li><a href="https://jestjs.io/">Jest test</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Here are some key features of the application :
+- **Driver categories fetch by api**
+- **Drivers stats component fetch by api**
+- **Use React Router to make the routes work**
+- **Use Jest Snapshot for testing**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Demo <a name="live-demo"></a>
+
+https://f1-driverstandings-react.onrender.com/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Clone the repository by clicking on the 'Code' button and copy the link
+
+git@github.com:smunoz1988/metrics-react.git
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- npm install
+- npm start to run the server
+- npm test to run jest tests
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git@github.com:smunoz1988/metrics-react.git
+```
+
+### Install
+
+Install this project with:
+
+- npm install
+- npm start to run the server.
+
+### Usage
+
+To run the project, execute the following command:
+Open the index.html file in a web browser using live server
+
+<!--
+Example command:
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+Use npm test for the mock Jest
+
+Example command:
+
+To run the stylehint linter
+
+```sh
+  npx stylehint "**/*.{css,scss}"
+```
+
+To run the ESLint linter
+
+```sh
+  npx eslint .
+```
+
+### Deployment
+
+You can see the live version of this project using:
+
+https://f1-driverstandings-react.onrender.com/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Santiago Munoz**
+
+- GitHub: [@smunoz](https://github.com/smunoz1988)
+- Twitter: [@twitterhandle](https://twitter.com/Santiag24209785)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/santiago-munoz-0b2b1a260)
+
+App design by:
+
+👤 Nelson Sakwa.
```

Comment:
- [ ] Please remove this acknowledgement of Nelson's design from the Author section and add it to the acknowledgement section where it is supposed to be :+1: .